gravitee-am-management-api/gravitee-am-management-api-service/src/main/java/io/gravitee/am/management/service/impl/IdentityProviderManagerImpl.java

```diff
@@ -62,6 +69,7 @@
  */
 @Component
 public class IdentityProviderManagerImpl extends AbstractService<IdentityProviderManager> implements IdentityProviderManager, EventListener<IdentityProviderEvent, Payload> {
+    private static final Set SUPPORTED_PASSWORD_ENCODER = Set.of("BCrypt", "SHA", "SHA-1", "SHA-256", "SHA-384", "SHA-512", "MD5", "SHA-256+MD5", "None");
```

Comment:
should we allow "None"?

Reply:
maybe not indead and I have also some doubt about MD5 and SHA-1 but they are avaible through the UI
I will remove the weakest ones